src/app/api/stripe/webhooks/route.ts

```
@@ -13,9 +14,22 @@
   const sig = headerPayload.get('stripe-signature') as string
   const body = await req.text()
 
-  const evt = stripe.webhooks.constructEvent(body, sig, STRIPE_WEBHOOK_SECRET)
+  try {
+    const evt = await stripe.webhooks.constructEventAsync(body, sig, STRIPE_WEBHOOK_SECRET)
 
-  console.log(evt.type)
+    switch (evt.type) {
+      case 'checkout.session.completed':
+        const { metadata } = evt.data.object
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)